recipes/_compute_base_config.rb

```diff
@@ -81,28 +81,30 @@
 
 # Parse shared directory info and turn into an array
 shared_dir_array = node['cfncluster']['cfn_shared_dir'].split(',')
-shared_dir_array.each_with_index do |dir, index|
-  shared_dir_array[index] = dir.strip
-  shared_dir_array[index] = "/" + shared_dir_array[index] unless shared_dir_array[index].start_with?("/")
-end
 
 # Mount each volume with NFS
-shared_dir_array.each do |dirname|
-  # Created shared mount point
-  directory dirname do
-    mode '1777'
-    owner 'root'
-    group 'root'
-    recursive true
-    action :create
-  end
+shared_dir_array.each do |dir|
```

Comment:
up to you: we could avoid to define `dir` and just use `dirname` by replacing the value everywhere.